Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -145,6 +153,8 @@ private class CustomCell: UITableViewCell {
 
         innerStroke.frame = background.frame
 
+        closeTab.frame = CGRect(x: backgroundHolder.frame.width - margin, y: 2, width: 12, height: 12)
```

Comment:
I'd like us to transition this file to use autolayout instead of frames if possible, but I suppose setting the frame is fine for now.

A couple comments about this line:
 1. Can you add constants to `TabTrayControllerUX` called `CloseButtonMargin` and `CloseButtonSize`, then use those here? I think it would look like this:
 ```
closeTab.frame = CGRect(x: backgroundHolder.frame.width - TabTrayControllerUX.CloseButtonSize - TabTrayControllerUX.CloseButtonMargin,
        y: TabTrayControllerUX.CloseButtonMargin,
        width: TabTrayControllerUX.CloseButtonSize,
        height: TabTrayControllerUX.CloseButtonSize)
```
A bit more verbose, I know, but it's good to have these values extracted into constants. That helps to make sure the margins are equal, for example.

 1. Can you increase the button size and margins -- maybe 15 and 10, respectively, so the button is centered and a bit larger? @darrinhenein and @tecgirl may be able to provide exact values (in addition to an alternate close image).

Also, we'll probably want the button to have edge insets set so the tap area is bigger than the visible "x" (since the button is fairly small).